### ISSUE TITLE
Tabbing in PDK Date Component not working for Firefox

### ISSUE THREAD

**Issue by justin-r-oliver** [CJSCommonPlatform/govuk_single_page_pdk#139]
The PDK 'pdk-date-input' component does not allow a user to tab out of the component. This is only seen in Firefox. This means that page using this component will fail accessibility testing. 

Firefox Version: 54.0.1 
PDK Version: v0.1.75 
Example: http://cjscommonplatform.github.io/govuk_single_page_pdk/#/components/forms-and-errors#date-input

Logged here: https://tools.hmcts.net/jira/browse/PE-819
